### PR TITLE
fix: Volume size is ingored

### PR DESCRIPTION
**Pull request by npalm**
The problen is most likely introduces in one of the PR's to support multiple ebs.

The following options for volumes are now supported.
- Default, create an ebs for amazon linux
- Provide explicit ebs mappings via block_device_mappings
- No ebs, set block_device_mappings to []
- Deprecated! Overwrite the default volume size by setting volume_size

close: #1954

### CODE REVIEW

**Review by JeroenKnoops — APPROVED** [review]
LGTM

File: modules/runners/main.tf [hunk 1]
```diff
@@ -64,7 +64,7 @@ resource "aws_launch_template" "runner" {
       ebs {
         delete_on_termination = lookup(block_device_mappings.value, "delete_on_termination", true)
```
Comment (ScottGuymer) [inline modules/runners/main.tf:65]: I think we should take the opportunity to remove this complexity here and just use the array of block mappings and remove the lookups.

Comment (npalm) [reply modules/runners/main.tf:65]: Shall we do this in a separate PR. And combine that PR with the breaking changes in #2013

Comment (ScottGuymer) [reply modules/runners/main.tf:65]: Can we make use of https://www.terraform.io/language/functions/defaults to help with this?

Comment (npalm) [reply modules/runners/main.tf:65]: This feature is still experimental, so for that reason I was not using them yet.

Comment (npalm) [reply modules/runners/main.tf:65]: In this PR only trying to fix broken functionality

Comment (ScottGuymer) [reply modules/runners/main.tf:65]: ok np

**Review by ScottGuymer — APPROVED** [review]
:shipit:

**Pull request merged**